Strings.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RedBallTracker
+{
+    public static class Strings
```

Comment:
Maybe this class could be called constants and also hold other types constants if needed?

Reply:
That's actually a great idea, I will change this